alicloud/service_alicloud_cms.go

```diff
@@ -69,3 +71,16 @@ func (s *CmsService) WaitForCmsAlarm(id string, enabled bool, timeout int) error
 	}
 	return nil
 }
+
+func (s *CmsService) BuildJsonWebhook(webhook string) string {
+	return fmt.Sprintf("{\"method\":\"post\",\"url\":\"%s\"}", webhook)
+}
+
+func (s *CmsService) ExtractWebhookFromJson(webhookJson string) (string, error) {
```

Comment:
I think the method `ExtractWebhookFromJson ` is needless and it just a `Unmarshal ` action.

Reply:
that is true. still there is some code involved for umarshaling. This is why I have put it into a separate  function to make the code more clear and easier to understand. OK for you?